Include/internal/pycore_frame.h

```diff
@@ -52,6 +52,7 @@ enum _frameowner {
     FRAME_OWNED_BY_GENERATOR = 1,
     FRAME_OWNED_BY_FRAME_OBJECT = 2,
     FRAME_OWNED_BY_CSTACK = 3,
+    FRAME_OWNED_BY_ACCIDENT = 4,
```

Comment:
Maybe you should add a NEWS describing that you added a new value to the `enum_frameowner` enumeration in `pycore_frame.h`.

I think that making modifications to the interpreter core requires adding.

Reply:
This is not part of the public API; it’s an internal fix that has no direct relevance to Python users or extension authors.

However, the surrounding code has changed, so I’m closing this in favor of

- #124148